### ISSUE TITLE
[A11y] ARIA 1.3: Support role="image"

### ISSUE THREAD

**Issue by bsmth** [mdn/content#27750]
## Acceptance Criteria

- [x] The listed features are documented sufficiently on MDN
- [ ] BCD is updated
- [ ] Interactive example and data repos are updated if appropriate
- [x] The content has been reviewed as needed

### For folks helping with Firefox related documentation

- [x] Set bugs to `dev-doc-complete`
- [x] Add entry to Firefox release notes if feature is enabled in release
- [x] Add entry to Firefox experimental features page if feature is not yet enabled in release

## Features to document

>Added in ARIA 1.3: https://github.com/w3c/aria/pull/1370
image is now the primary ARIA role, with img being a synonym

__wpt:__ https://wpt.fyi/results/wai-aria/role/synonym-roles.html?label=master&label=experimental&aligned

## Related Gecko bugs

- https://bugzilla.mozilla.org/show_bug.cgi?id=1829269

## Other

- [x] Check content open issues to see if any pertain to the subject matter. If there are any that can be closed because of the work, do so. If there are any that can be fixed relatively quickly because of the knowledge from completing this issue and you have time, feel free to go ahead and fix them.
- [x] Check if glossary updates are required for the feature you're documenting - whether an existing term needs to be updated or a new term should be added.
- [ ] Check if BCD update means that content pages need to have experimental markup removed or deprecated markup added (front matter tags and macros).

__Related issues and pull requests:__
- [x] Relnote: https://github.com/mdn/content/pull/27926
- [x] Content: TBD whether deprecation notice & renaming is called for
-  <strike>BCD: TODO</strike> No BCD existing for ARIA roles exist

**Comment by bsmth** [comment]
Adding a follow-up for renaming `img` to `image` with a deprecation warning for the older role in https://github.com/mdn/content/issues/28170
